src/Tactics.lidr

```diff
@@ -1021,6 +1017,10 @@ Now suppose that we want to convince Idris of the (rather obvious) fact that
 the proofs we did with \idr{sillyfun} above, it is natural to start the proof
 like this:
 
+\todo[inline]{Make this \idr{total}, or at least describe why it's not in its
+current implementation. Should we introduce \idr{Decidable.Equality} here?}
+
+> partial
```

Comment:
The "explicit implicits" here are unfortunate, maybe you can make it work without those.